owncloudApp/src/main/java/com/owncloud/android/presentation/ui/settings/fragments/SettingsSecurityFragment.kt

```diff
@@ -56,28 +56,22 @@ class SettingsSecurityFragment : PreferenceFragmentCompat() {
     private val enablePasscodeLauncher =
         registerForActivityResult(ActivityResultContracts.StartActivityForResult()) { result ->
             if (result.resultCode != Activity.RESULT_OK) return@registerForActivityResult
-            val passcodeEnableResult = securityViewModel.handleEnablePasscode(result.data)
-            if (passcodeEnableResult.isSuccess) {
+            else {
```

Comment:
The `else` clause could be omitted, couldn't it? In case `result.resultCode != Activity.RESULT_OK`, it will return, so it won't reach this code. Anyway, I think that it adds readability, so it's fine 👍

Reply:
Yes, it's not necessary, but as you said, I added it because of readability, I think it's easier this way